xml/System/AppDomainSetup.xml

```diff
@@ -383,12 +383,16 @@
       </Docs>
     </Member>
     <Member MemberName="ApplicationBase">
-      <MemberSignature Language="C#" Value="public string ApplicationBase { get; set; }" />
+      <MemberSignature Language="C#" Value="public string ApplicationBase { get; }" FrameworkAlternate="netcore-3.0" />
```

Comment:
this makes me think that we'll need different summaries for these APIs depending on the version. ECMAXML doesn't support that right now, correct?

Reply:
Yes that's correct ... there's no support currently for framework/moniker specific summaries. From what I've seen, the only recourse here is just to call it out manually in the written text.

Reply:
Yep, the same way we do for exceptions today. Should I open a customer suggestion for this?

Reply:
Yes I think it's something that we'll have to tackle eventually for sure ... a customer suggestion would give us something to discuss and investigate :)